bin/make_paper_titles.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python
+
+import json
+import sys
+import re
+
+with open(sys.argv[1]) as f:
+    for num,data in json.load(f).items():
+        if re.match(r'^P\d\d\d\dR\d+$', num):
```

Comment:
No `N` papers?

Reply:
For new issues the only N-papers we link to are the working drafts, but for older issues it would be good. I'll add it.